zulip/integrations/zephyr/zephyr_mirror_backend.py

```diff
@@ -759,12 +770,13 @@ def maybe_forward_to_zephyr(message: Dict[str, Any]) -> None:
 def zulip_to_zephyr(options: int) -> None:
     # Sync messages from zulip to zephyr
     logger.info("Starting syncing messages.")
+    backoff = RandomExponentialBackoff(timeout_success_equivalent=120)
```

Comment:
I also note `timeout_success_equivalent` is undocumented; do you understand what it's for?

I guess 4e02777aaa9e788843f581868291e8c4b3696ace explains it pretty clearly, so maybe we just want to move some of that commit message into the comments.

And I think 120 is a good value for that here.